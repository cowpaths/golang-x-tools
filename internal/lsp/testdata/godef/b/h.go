package b

func _() {
	// variable of type a.A
	var _ A //@mark(AVariable, "_"),hoverdef("_", AVariable)

	AStuff() //@hoverdef("AStuff", AStuff)
}
