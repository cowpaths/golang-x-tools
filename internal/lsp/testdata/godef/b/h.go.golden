-- AStuff-hoverdef --
```go
func AStuff()
```

[`a.AStuff` on pkg.go.dev](https://pkg.go.dev/github.com/cowpaths/golang-x-tools/internal/lsp/godef/a?utm_source=gopls#AStuff)
-- AVariable-hoverdef --
```go
var _ A
```

variable of type a\.A
